src/summary/level-two-routes.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { match, Switch, Route, useRouteMatch } from "react-router";
+import { AllergyCardLevelTwo } from "./history/allergy-card-level-two.component";
+
+export function LevelTwoRoutes(props: LevelTwoRoutesProps) {
+  let { path } = useRouteMatch();
```

Comment:
I don't think you need to call useRouteMatch, since you already have the `match` prop. I'm pretty sure that `props.match.url` is the exact same thing as `path`